server/pxf-hdfs/src/main/java/org/greenplum/pxf/plugins/hdfs/orc/ORCVectorizedMappingFunctions.java

```diff
@@ -66,6 +75,91 @@
         return result;
     }
 
+    /**
+     * Handles all the different orc list types that PXF supports
+     * @param batch the column batch to be processed
+     * @param columnVector this should be the ListColumnVector that contains another ColumnVector containing the actual data
+     * @param oid the GPDB oid mapping for this particular list
+     * @return returns a list of OneFields
+     */
```

Comment:
```suggestion
    /**
     * Serializes ORC lists of PXF-supported primitives in to Postgres array syntax
     * @param batch the column batch to be processed
     * @param columnVector this should be the ListColumnVector that contains another ColumnVector containing the actual data
     * @param oid the destination GPDB column OID
     * @return returns an array of OneFields, containing Strings
     */
```

Nitpick: the JavaDoc should try to capture the "contract" of this method that isn't captured in the types. Since `OneField` contains an `Object`, what handling means could be anything. The proposed edit is meant more as a suggestion of ways you could add more detail.